File: app/src/main/java/com/neome/java/api/core/base/dto/DtoDevice.java
// neome.ai API - do not change
//

package com.neome.java.api.core.base.dto;

import com.neome.java.api.meta.base.Types.DeviceId;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class DtoDevice
{
  public Date creationTime;

  public DeviceId deviceId;

  @Nullable
  public String deviceName;

  @Nullable
  public Boolean isCurrentDevice;

  @Nullable
  public Boolean isOnline;

  @Nullable
  public Date lastOnlineTime;

  public Object state;
}

File: app/src/main/java/com/neome/api/core/base/dto/DtoDevice.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.base.dto

import com.neome.api.meta.base.Types.DeviceId

interface DtoDevice
{
  val creationTime: String
  val deviceId: DeviceId
  val deviceName: String?
  val isCurrentDevice: Boolean?
  val isOnline: Boolean?
  val lastOnlineTime: String?
  val state: Object
}